src/finufft.cpp

```diff
@@ -195,25 +196,23 @@ void onedim_fseries_kernel(BIGINT nf, FLT *fwkerhalf, spread_opts opts)
     a[n] = exp(2*PI*IMA*(FLT)(nf/2-z[n])/(FLT)nf);  // phase winding rates
   }
   BIGINT nout=nf/2+1;                   // how many values we're writing to
-  int nt = min(nout,(BIGINT)MY_OMP_GET_MAX_THREADS());      // how many chunks
+  int nt = min(nout,(BIGINT)opts.nthreads);         // how many chunks
   std::vector<BIGINT> brk(nt+1);        // start indices for each thread
   for (int t=0; t<=nt; ++t)             // split nout mode indices btw threads
     brk[t] = (BIGINT)(0.5 + nout*t/(double)nt);
-#pragma omp parallel
+#pragma omp parallel num_threads(nt)
```

Comment:
This is great.

Reply:
You mean, you like explicit num_threads throughout? So do I :)

BTW, I did ad hoc speed comparisons on the i7/GCC of schedule(static) vs schedule(dynamic,1) vs dynamic,10000, etc.   There isn't much logic to it, but explains some of my changes.

Reply:
Yes. This feels very much like the Right Thing to do…

Reply:
:)